### PR TITLE
Refactor: remove unused views from EDC and update catalog view

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
## Description

- Removed connector View and ExternalConnector classes. (#129)
- Removed Order View
- Updated Catalog View to work. (#131)
- Added edc controller to be secured by API KEY

Please note that actual code change is smaller because seven files with each 20 lines of license header and some documentation have been removed

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: frontend/src/views/NegotiationView.vue [hunk 1]
```diff
@@ -21,43 +21,27 @@
...
+      backendURL: import.meta.env.VITE_BACKEND_BASE_URL,
       catalog: {},
     };
   },
   methods: {
-    getCatalog() {
-      let vm = this;
-      fetch(vm.baseUrl + "/edc/negotiations")
+    getNegotiations() {
+      fetch(this.backendURL + "/edc/negotiations", {
```
Comment (eschrewe) [inline frontend/src/views/NegotiationView.vue:32]: With the default settings, this.backendURL has a trailing slash. Therefore a leading slash in front of "/edc/negotiations" is not working correctly.

File: frontend/src/views/NegotiationView.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!--
- Copyright (c) 2022,2023 Volkswagen AG
- Copyright (c) 2022,2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
- Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+ Copyright (c) 2022,2024 Volkswagen AG
```
Comment (eschrewe) [inline frontend/src/views/NegotiationView.vue:2]: Better: 2022-2024

File: frontend/src/router/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright (c) 2022,2023 Volkswagen AG
- * Copyright (c) 2022,2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
- * Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2022,2024 Volkswagen AG
```
Comment (eschrewe) [inline frontend/src/router/index.js:2]: Better: 2022-2024

File: frontend/src/views/CatalogView.vue [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <!--
- Copyright (c) 2022,2023 Volkswagen AG
- Copyright (c) 2022,2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
- Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+ Copyright (c) 2022,2024 Volkswagen AG
```
Comment (eschrewe) [inline frontend/src/views/CatalogView.vue:2]: Better: 2022-2024

**Review by eschrewe — CHANGES_REQUESTED** [review]
Thank you!
Only some minor remarks from my side.

File: frontend/src/views/NegotiationView.vue (hunk 1 above)
Comment (tom-rm-meyer-ISST) [reply frontend/src/views/NegotiationView.vue:32]: fixed

**Review by mhellmeier — APPROVED** [review]
Fine so far, the conflicts have to be solved.

**Pull request merged**